optaplanner-core/src/main/java/org/optaplanner/core/api/stat/ScoreCalculationSpeedSubSingleStatisticListener.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.api.stat;
+
+import org.optaplanner.core.impl.phase.event.PhaseLifecycleListenerAdapter;
+import org.optaplanner.core.impl.phase.scope.AbstractPhaseScope;
+import org.optaplanner.core.impl.phase.scope.AbstractStepScope;
+import org.optaplanner.core.impl.solver.scope.SolverScope;
+
+public class ScoreCalculationSpeedSubSingleStatisticListener<Solution_> extends PhaseLifecycleListenerAdapter<Solution_> {
```

Comment:
I see that this code was taken directly from the benchmarker. That is understandable. But the `SubSingleStatistic` name is not, in this new context. There are no `SubSingle`s in core, and therefore this name needs to change.

Reply:
Hibernate does use Statistics for its Stat tracker: https://github.com/hibernate/hibernate-orm/blob/main/hibernate-micrometer/src/main/java/org/hibernate/stat/HibernateMetrics.java

So perhaps `ScoreCalculationCountStatistic`?

Reply:
Works for me.